packages/admin/cms-admin/src/dam/Table/FolderTableRow.tsx

```diff
@@ -62,13 +66,25 @@ export const FolderTableRow: React.FunctionComponent<FolderTableRowProps> = ({
     allowedMimetypes,
     archived,
     hideMultiselect,
+    isNew = false,
+    scrollIntoView = false,
 }) => {
     const multiselectApi = useDamMultiselectApi();
     const client = useApolloClient();
     const { allAcceptedMimeTypes } = useDamAcceptedMimeTypes();
     const { moveItem } = useDamDnD();
 
     const [isHovered, setIsHovered] = React.useState<HoverStyle>();
+    const [markAsNew, setMarkAsNew] = React.useState<boolean>(false);
+
+    React.useEffect(() => {
+        if (isNew) {
+            setMarkAsNew(true);
+            setTimeout(() => {
```

Comment:
You need to clear the timeout in the clean up function of the useEffect: https://www.alexhughes.dev/blog/setTimeout-with-hooks/